src/cr/cube/matrix.py

```diff
@@ -667,6 +712,10 @@ def columns(self):
             for counts, base_counts, element, table_margin in self._column_generator
         )
 
+    @lazyproperty
+    def _mr_shadow_proportions(self):
```

Comment:
add a docstring explaining what a shadow proportion is.

Reply:
@malecki can u provide a meaningful description for what a shadow prop is?

Reply:
```suggestion
    def _mr_shadow_proportions(self):
        """
        Cube containing item-wise selections, overlap, and nonoverlap 
        with all other items in a multiple response dimension, for each 
        element of any prepended dimensions: 
        A 1d interface to a 4d hypercube of underlying counts.
        """
```